test/new_relic/agent/llm/chat_completion_message_test.rb

```diff
@@ -0,0 +1,78 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative '../../../test_helper'
+
+module NewRelic::Agent::Llm
+  class ChatCompletionMessageTest < Minitest::Test
+    def test_attributes_assigned_by_parent_present
+      assert_includes NewRelic::Agent::Llm::ChatCompletionMessage.ancestors, NewRelic::Agent::Llm::LlmEvent
+      assert_includes NewRelic::Agent::Llm::LlmEvent::AGENT_DEFINED_ATTRIBUTES, :transaction_id
+
+      in_transaction do |txn|
+        event = NewRelic::Agent::Llm::ChatCompletionMessage.new
+
+        assert_equal txn.guid, event.transaction_id
+      end
+    end
+
+    def test_attributes_in_parent_list_can_be_assigned_on_init
+      assert_includes NewRelic::Agent::Llm::LlmEvent::ATTRIBUTES, :id
+
+      event = NewRelic::Agent::Llm::ChatCompletionMessage.new(id: 123)
+
+      assert_equal 123, event.id
+    end
+
+    def test_included_module_attributes_list_can_be_assigned_on_init
+      assert_includes NewRelic::Agent::Llm::ChatCompletionMessage.ancestors, NewRelic::Agent::Llm::ChatCompletion
+      assert_includes NewRelic::Agent::Llm::ChatCompletion::ATTRIBUTES, :conversation_id
+
+      conversation_id = '123abc'
+      event = NewRelic::Agent::Llm::ChatCompletionMessage.new(conversation_id: conversation_id)
+
+      assert_equal conversation_id, event.conversation_id
+    end
+
+    def test_attributes_constant_values_can_be_passed_as_args_and_set_on_init
+      assert_includes NewRelic::Agent::Llm::ChatCompletionMessage::ATTRIBUTES, :role
+      role = 'user'
+      event = NewRelic::Agent::Llm::ChatCompletionMessage.new(role: role)
+
+      assert_equal role, event.role
+    end
+
+    def test_args_passed_to_init_not_set_as_instance_vars_when_not_in_attributes_constant
+      event = NewRelic::Agent::Llm::ChatCompletionMessage.new(fake: 'fake')
+
+      refute_includes event.attributes, :fake
+      refute event.instance_variable_defined?(:@fake)
+    end
+
+    def test_record_creates_an_event
```

Comment:
Tomorrow I'll update the `test_record_creates_an_event` tests include values for every attributes on the event to make sure they're defined